### PR TITLE
ID-1858: Fix issue in sorting risk score in table template.

### PR DESCRIPTION
**Pull request by jbabac**
Demo:
https://www.loom.com/share/58f2091bd52d47e8af3860f74bbe645e

Changes:
1. Fixed issues in sorting risk_score as well as the count in table template.
2. Made the risk_score to be sorted in descending by default.

Screenshots:
image
image

### CODE REVIEW

File: domaintools_iris_connector.py [hunk 1]
```diff
@@ -251,6 +252,22 @@ def _do_query(self, service, action_result, query_args=None):
                 e,
             )
 
+    def _convert_risk_scores_to_string(self, results_data):
+        # We need this to make the table view sortable.
+        # For some unknown reason, numeric values are not sortable using
+        # the default table view from Splunk SOAR template.
+        final_result = []
+        for result in results_data:
+            result.get("domain_risk").update({"risk_score_string": self._convert_null_value_to_empty_string(result.get("domain_risk", {}).get("risk_score"))})
```
Comment (briluza) [inline domaintools_iris_connector.py:261]: lgtm! one comment though but more on a pedantic change is that can we shorten the line from 158 chars to max 100 for more readability? 

e.g. assigning the result of self._convert_risk_scores_to_string to a variable.

File: domaintools_iris_connector.py [hunk 2]
```diff
@@ -378,23 +395,32 @@ def _reverse_lookup_domain(self, param):
...
+        sorted_ips = sorted(
+            ips,
+            key=lambda d: 0 if d.get("count_string") == "" else int(d.get("count_string")),
+            reverse=True)
+        action_result.update_summary({"ip_list": sorted_ips})
 
         return action_result.get_status()
 
+    @staticmethod
+    def _convert_null_value_to_empty_string(value):
```
Comment (briluza) [inline domaintools_iris_connector.py:421]: can it be a normal private function instead making it a staticmethod? as we only use it inside the class.

**Comment by briluza** [comment]
Overall, LGTM! ✅

File: domaintools_iris_connector.py (hunk 1 above)
Comment (jbabac) [reply domaintools_iris_connector.py]: done

**Pull request merged**